seatunnel-apis/seatunnel-api-flink/src/main/java/org/apache/seatunnel/flink/FlinkEnvironment.java

```diff
@@ -322,4 +325,18 @@ private void setCheckpoint() {
         }
     }
 
+    public void registerResultTable(Config config, String tableName, DataStream<Row> dataStream) {
```

Comment:
tableName always be `RESULT_TABLE_NAME`, so you can remove it.

Reply:
Done.